src/components/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from "react";
+
+interface IProps {
+  inverted?: boolean;
+  text?: string;
+  color?: boolean;
+  size?: boolean;
+}
```

Comment:
I think text props is required and other props need default value

Reply:
done